### PR TITLE
INT-B-20043

### PR DESCRIPTION
**Pull request by loganwc**
## Summary

Blocking SAFETY moves from office users without SAFETY privileges

## How to test
1. Create a safety move
2. Login as an office user with safety privileges
3. Copy the safety move url
4. Login as an office user without safety privileges
5. Try to navigate to safety move with url
6. Do not see the page

### CODE REVIEW

File: pkg/handlers/ghcapi/move.go [hunk 1]
```diff
@@ -66,8 +72,13 @@ func (h GetMoveHandler) Handle(params moveop.GetMoveParams) middleware.Responder
 				}
 			}
 
-			payload := payloads.Move(move)
-			return moveop.NewGetMoveOK().WithPayload(payload), nil
+			if move.Orders.OrdersType == "SAFETY" && !privileges.HasPrivilege(models.PrivilegeTypeSafety) {
```
Comment (pambecker) [inline pkg/handlers/ghcapi/move.go:75]: OrdersType is blank for me here

**Review by pambecker — CHANGES_REQUESTED** [review]

**Review by pambecker — APPROVED** [review]

**Review by pambecker — APPROVED** [review]
Code works as expected.  Good to go once the tests pass

**Comment by paulstonebraker** [comment]
Is PO okay with the "oops something went wrong screen"? I thought they had requested the "404 can't find the page youre looking for" screen instead

**Comment by loganwc** [comment]
> Is PO okay with the "oops something went wrong screen"? I thought they had requested the "404 can't find the page youre looking for" screen instead

that's the screen you get if you 404, try to navigate to fake locator and it'll just show the oops something went wrong screen

**Comment by paulstonebraker** [comment]
> > Is PO okay with the "oops something went wrong screen"? I thought they had requested the "404 can't find the page youre looking for" screen instead
> 
> that's the screen you get if you 404, try to navigate to fake locator and it'll just show the oops something went wrong screen

Yeah I don't have a horse in the race, just thought I remembered them requesting the 404 page.

**Review by paulstonebraker — APPROVED** [review]

**Review by pambecker — APPROVED** [review]

**Pull request merged**